test/dynamo/test_dynamo.py

```diff
@@ -245,22 +245,22 @@ def fn_fallback(t):
     cpu_res = fn_fallback(t)
     xla_dynamo_res = dynamo_fn(t_xla)
     self.assertTrue(torch.allclose(cpu_res, xla_dynamo_res.cpu()))
-    self.assertEqual(met.metric_data('CompileTime')[0], 4)
-    self.assertEqual(met.metric_data('ExecuteTime')[0], 8)
+    self.assertEqual(met.metric_data('CompileTime')[0], 3)
```

Comment:
The `CompileTime` and `ExecuteTime` only differs for this `test_fallback_multiple_submodules` unit test. With `allows_single_node_partition=True`, the partition is different and sees slightly different `CompileTime` and `ExecuteTime` metrics.